tests/mypy_test.py

```diff
@@ -336,7 +321,7 @@ def main():
             if not is_probably_stubs_folder(distribution, distribution_path):
                 continue
 
-            if not is_supported(distribution_path, major):
+            if major == 2:
```

Comment:
It would be cleaner to put the whole loop inside `if major != 2`. This way it won't print `Running mypy ...` when it isn't actually going to run mypy.